setup/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "setup-notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Here's an example of a build script:
`ncc build index.ts -o dist --source-map --minify --license licenses.txt`

---

Should that be added into the scripts section within the value of the test property?

---

> Should that be added into the scripts section within the value of the test property?

Oh, it's not for test. Should be something like this:
```suggestion
  "scripts": {
      "build": "ncc build index.ts -o dist --source-map --minify --license licenses.txt",
      "test": "echo \"Error: no test specified\" && exit 1"
   }
```

---

Oh! that makes life so much easier! Clearly, my lack of JS knowledge is showing. :) Thank you @Two-Hearts

---

> Oh! that makes life so much easier! Clearly, my lack of JS knowledge is showing. :) Thank you @Two-Hearts

No problem @duffney. I'm also in the learning phase :)